app/src/main/java/com/github/polybooks/FillSaleActivity.kt

```diff
@@ -112,8 +128,17 @@ class FillSaleActivity : AppCompatActivity(), AdapterView.OnItemSelectedListener
         findViewById<TextView>(R.id.filled_publish_date).apply {
             text = dateFormat.format(book.publishDate!!.toDate()) ?: ""
         }
-        // TODO whole lines could be removed from UI when argument is null instead of placeholding with default value
+        // TODO whole lines could be removed from UI (visibility = View.GONE) when argument is null instead of placeholding with default value
         findViewById<TextView>(R.id.filled_format).apply { text = book.format ?: "" }
+
+        if (pictureFileName.isNotEmpty()) {
+            Log.w("BookFuture", this.filesDir.path + '/' + pictureFileName)
+            val bmImg = BitmapFactory.decodeFile(this.filesDir.path + '/' + pictureFileName)
+            //TODO The picture gets rotated 90°, could work on fixing this, but not urgent.
+            findViewById<ImageView>(R.id.proof_picture).setImageBitmap(bmImg)
```

Comment:
Is this impossible to test?

Reply:
It is impossible to test as long as the camera can't be tested. It would required Cirrus to be able to take **and** save a picture to memory.